src/cli/ImportFeedCommand.ts

```diff
@@ -159,10 +153,40 @@ export class ImportFeedCommand implements CLICommand {
     const file = this.getFeedFile(filename);
     const tables = await this.tables(file);
     const tableStream = new MySQLStream(filename, file, tables);
-    const stream = readFile(path + filename).pipe(tableStream);
 
     try {
-      await streamToPromise(stream);
+      await new Promise((resolve, reject) => {
+        const lineStream = new LineByLineReader(path + filename);
+
+        lineStream.on('line', async function (line: any) {
+          // Pause after every line to enforce correct line ordering in multi-record files.
+          lineStream.pause();
```

Comment:
Don't use `any` if the type is known.
line is of type Buffer.

Also not sure how many "lines" we're talking about here. You can also just accumulate lines into Buffer, and then let's say every 1000 lines or so write to the writable buffer.

Pausing the stream might cause the data to accumulate into the buffer anyway (thus using a lot of memory). Haven't tried that, but read about how it works though.

Reply:
It's fine, the code in line-by-line handles that correctly, and I'd rather have it handle the abstraction. It does its own abstraction of reading X bytes from a file, splitting into lines and feeding me line by line, so I don't think there's a need for an extra buffer.